unlock-app/src/services/storageService.js

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios'
+
+export default class StorageService {
+  constructor(host) {
+    this.HOST = host
```

Comment:
could we stick to lowercase? 
```suggestion
    this.host = host
```